tests/foreman/ui/test_gpgkey.py

```diff
@@ -384,7 +380,118 @@ def test_consume_content_1(self):
+        entities.Repository(id=repo_id).sync()
+        # step 4: Create content view
+        content_view = entities.ContentView(
+            organization=self.org_id
+        ).create_json()
+        # step 5: Associate repository to new content view
+        response = client.put(
+            entities.ContentView(id=content_view['id']).path(),
+            {u'repository_ids': [repo_id]},
+            auth=get_server_credentials(),
+            verify=False,
+        )
+        response.raise_for_status()
+        # step 6: Publish content view
+        entities.ContentView(id=content_view['id']).publish()
+        # step 6.2: Promote content view to lifecycle_env
+        content_view = entities.ContentView(id=content_view['id']).read_json()
+        self.assertEqual(len(content_view['versions']), 1)
+        entities.ContentViewVersion(
+            id=content_view['versions'][0]['id']
+        ).promote(lifecycle_env['id'])
+        # step 7: Create activation key
+        ak_id = entities.ActivationKey(
+            name=activation_key_name,
+            environment=lifecycle_env['id'],
+            organization=self.org_id,
+            content_view=content_view['id'],
+        ).create_json()['id']
+        for subscription in entities.Organization(
+                id=self.org_id).subscriptions():
+            if subscription['product_name'] == product_name:
+                entities.ActivationKey(id=ak_id).add_subscriptions({
+                    'quantity': 1,
+                    'subscription_id': subscription['id'],
+                })
+                break
+        # Create VM
+        package_name = "cow"
+        with VirtualMachine(distro='rhel66') as vm:
+            # Download and Install rpm
+            result = vm.run(
+                "wget -nd -r -l1 --no-parent -A '*.noarch.rpm' http://{0}/pub/"
+                .format(conf.properties['main.server.hostname'])
+            )
+            self.assertEqual(
+                result.return_code, 0,
+                "failed to fetch katello-ca rpm: {0}, return code: {1}"
+                .format(result.stderr, result.return_code)
+            )
+            result = vm.run(
+                'rpm -i katello-ca-consumer*.noarch.rpm'
+            )
+            self.assertEqual(
+                result.return_code, 0,
+                "failed to install katello-ca rpm: {0} and return code: {1}"
+                .format(result.stderr, result.return_code)
+            )
+            # Register client with foreman server using activation-key
+            result = vm.run(
+                u'subscription-manager register --activationkey {0} '
+                '--org {1} --force'
+                .format(activation_key_name, self.org_label)
+            )
+            '''self.assertEqual(
+                result.return_code, 0,
+                "failed to register client:: {0} and return code: {1}"
+                .format(result.stderr, result.return_code)
+            )'''
```

Comment:
Or can we turn this in to a regular Python comment?

    # self.assertEqual(
    #     result.return_code, 0,
    #     'Failed to register client {0}. Return code: {1}'
    #     .format(result.stderr, result.return_code)
    # )